src/main/java/entity/Info.java

```diff
@@ -10,9 +10,9 @@
  */
 public class Info {
     private final String id;
-    private final String name;
-    private final String description;
-    private final String category;
+    private String name;
+    private String description;
+    private String category;
```

Comment:
```suggestion
    private final String name;
    private final String description;
    private final String category;
```
Entities must be immutable. These fields should remain ```final``` to maintain data integrity and prevent unauthorized modifications.